pkg/cloud/vsphere/context/machine_context.go

```diff
@@ -174,11 +174,33 @@ func (c *MachineContext) IPAddr() string {
 	if c.Machine == nil {
 		return ""
 	}
+
+	var err error
+	var preferredAPIServerCIDR *net.IPNet
+	if c.MachineConfig.MachineSpec.Network.PreferredAPIServerCIDR != "" {
+		_, preferredAPIServerCIDR, err = net.ParseCIDR(c.MachineConfig.MachineSpec.Network.PreferredAPIServerCIDR)
+		if err != nil {
+			c.Logger.Error(err, "error parsing preferred apiserver CIDR")
+			return ""
+		}
+
+		c.Logger.V(2).Info("detected preferred apiserver CIDR", "preferredAPIServerCIDR", preferredAPIServerCIDR)
```

Comment:
Hi @andrewsykim,

Maybe V3 or V4? I agree with others we should figure out a balance to our log verbosity. I don't feel strongly about this instance, however.